run_bamstrap

```diff
@@ -23,12 +23,9 @@ if ! command -v ruby >/dev/null || \
   eval "$(curl -fsSL https://raw.githubusercontent.com/Linuxbrew/install/master/install-ruby)"
 fi
 
-if [ -z "$GITHUB_TOKEN" ]; then
-  echo "Aborting! No GITHUB_TOKEN provided"
-  echo "1. Follow these instructions to generate a personal access token: https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line"
-  echo "2. Provide this token in the GITHUB_TOKEN environment variable when running this script"
-  exit 6
-fi
+
```

Comment:
Just to cover all bases, let's make it an additional request with a possible default.
```suggestion

if [ -z "$GITHUB_TOKEN" ]; then
```

Reply:
I don't think I understand this comment? And can we use a default value? It just wouldn't work when we curl?

Reply:
Ah, I mean, if they provide `GITHUB_TOKEN` as an ENV when running, this script, then use it, otherwise, ask for it.